assets/components/ctaLink/ctaLink.jsx

```diff
@@ -10,19 +10,26 @@ import Svg from 'components/svg/svg';
 
 type PropTypes = {
   text: string,
-  url: string,
+  url?: string,
+  onClick?: () => void,
 };
 
 
 // ----- Component ----- //
 
 export default function CtaLink(props: PropTypes) {
-
   return (
-    <a className="component-cta-link" href={props.url}>
+    <a className="component-cta-link" href={props.url} onClick={props.onClick}>
       <span>{props.text}</span>
       <Svg svgName="arrow-right-straight" />
     </a>
   );
 
 }
+
+
+CtaLink.defaultProps = {
+  url: undefined,
+  onClick: undefined,
```

Comment:
[Elsewhere](https://github.com/guardian/support-frontend/pull/97/files#diff-86bdd2cf34bec7b42dfaadb94a11585bR57) we've used `null` instead of `undefined`, can we standardise on this?